README.md

```diff
@@ -30,14 +30,14 @@ from python_liftbridge import Lift, Message, Stream, ErrStreamExists
 # Create a Liftbridge client.
 client = Lift(ip_address='localhost:9292', timeout=5)
 
-# Create a stream attached to the NATS subject "foo".
+# Create a stream attached to the NATS subject "foo" with the name "foo-stream".
 try:
```

Comment:
I think it would be just
`Create a Liftbridge stream with name "foo-stream"`

to avoid confusion